### ISSUE TITLE
Improvement: Cutting Edge Technology

### ISSUE THREAD

**Issue by Vulptrix** [terraforming-mars/terraforming-mars#5008]
Make an indicator for Cutting Edge Technology present.
My though about it is -2 with yellow/red border like cards requirement below the normal cost reducing.

**Comment by kberg** [comment]
Yeah, I'm thinking about it. Either this, or some button that shows all discounts.
